.eslintrc

```diff
@@ -1,22 +1,36 @@
 {
+  "parser": "babel-eslint",
+  "extends": "airbnb",
   "env": {
     "browser": true,
     "node": true
   },
-  ecmaFeatures: {
-    jsx: true
-  },
-  "globals": {
-    "window": true,
-    "document": true
+  "rules": {
```

Comment:
I never used them for my own projects and I never had any issues. Added these back in the latest commit.